src/client/components/reservations/AddReservation.js

```diff
@@ -0,0 +1,91 @@
+import React, { useContext } from "react";
+import { useEffect, useState } from "react";
+
+const AddReservation = (props) => {
+  const [phone, setPhone] = useState("");
+  const [email, setEmail] = useState("");
+  const [name, setName] = useState("");
+  const [guests, setGuests] = useState(1);
+  const [newReservation, setNewReservation] = useState({});
+
+  const clearForm = () => {
+    setPhone("");
+    setEmail("");
+    setName("");
+    setGuests(1);
+  };
+
+  const addNewReservation = async (event) => {
+    event.preventDefault();
+
+    try {
+      const data = await fetch("/api/reservations", {
+        method: "POST",
+        mode: "cors",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email: email,
+          phone: phone,
+          name: name,
+          meal_id: props.meal,
+          guests: guests,
```

Comment:
2 things here:

1 - You can use shorthand notation, as show below.
2 - I suggest that you set the `meal_id` in server since it should be unique, it should be set upon creation of the meal.


```suggestion
          email,
          phone,
          name,
          meal_id: props.meal,
          guests,
```

Reply:
Corrected the shorthand. 
Not sure about the meal_id. This is not meal creation,  but reservation for a specific meal, where the meal_id in the reservation is a foreign key to the id of a meal in meals table. Therefore, I am fetching and using the id that comes from the existing specific meal. 
I don't see how I can do this in server - I need to know the specific id of an existing meal and set it to the reservation.

Reply:
You can pass a meal_id when you do the booking in the backend and access the meal in the backend.

Anyway Just an observation that you can think about for the future.

Reply:
ohhhh then I don't know how to do that 🙈 
Here in frontend I easily fetch what the meal and it's id is, but I don't have an idea how I would dynamically fetch the meal_id in api/reservations POST from /meals, as I am having a reservation form fold-out at each available meal. 
In /meals:id, I figure I could just use params id, but in /meals .... 🤔